### PR TITLE
Add action message configuration options

### PR DESCRIPTION
**Pull request by zhen0**
## Summary

Add automation action message configuration options

## Changes

Updates automations docs

## Importance

Helps users understand how to customize automation notifications

## Checklist

This PR:

- [ ] adds new tests (if appropriate)
- [ ] adds a change file in the `changes/` directory (if appropriate)
- [ ] updates docstrings for any new functions or function arguments, including `docs/outline.toml` for API reference docs (if appropriate)

### CODE REVIEW

File: docs/orchestration/concepts/automations.md [hunk 1]
```diff
@@ -8,29 +8,212 @@ You can set up and edit your Automations using the [API](/orchestration/concepts
 
 ## Events
 
-An _event_ is a configurable condition that, when met, will cause an automation to run. Currently you can configure automations for the following events:
-- A flow run enters a given state or states
-- Runs from multiple (or all!) flows enter a given state or states
+The automations system is composed of events and actions. When an _event_ is paired with an _action_, we call it an _automation_.
```
Comment (zangell44) [inline docs/orchestration/concepts/automations.md:11]: This is a very nice definition. I think it belongs in the Automation section above.

**Review by zangell44 — CHANGES_REQUESTED** [review]
Minor comment

File: docs/orchestration/concepts/automations.md (hunk 1 above)
Comment (madkinsz) [reply docs/orchestration/concepts/automations.md]: Thanks! Updated the intro section

**Comment by madkinsz** [comment]
Rendered at https://deploy-preview-4553--prefect-docs.netlify.app/orchestration/concepts/automations.html#automations

**Review by madkinsz — APPROVED** [review]

**Pull request merged**